### ISSUE TITLE
Installing with pandas '0.22.0'

### ISSUE THREAD

**Issue by noobmaster29** [robertmartin8/PyPortfolioOpt#127]
**Operating system, environment, python version**
Mac, Python 3.5, pandas 0.22.0

**Error message**
I wanted to ask if it is possible to install and run this with Pandas 0.22.0. I see in requirements that pandas>=0.25.3. I assume that if I pip this directly, pip will just force pandas to update and that is not what I want. 

I am still running pandas 0.22.0 because some other things I need to run depends on panels to work and I haven't found a way to replace this. 

Please let me know. Thanks in advance.

**Comment by noobmaster29** [comment]
Ok, I've

**Comment by noobmaster29** [comment]
Hmmm, not sure what I pressed to close the issue. 

I've clone the repository and just imported it within the directory I'm working in. It seems that pypfopt can run with pandas 0.22.0. 

pypfopt.__version__
'1.2.2'
pd.__version__
'0.22.0'
np.__version__
'1.14.6

I guess this is a workaround for me for now. :)

**Comment by robertmartin8** [comment]
Hi,

Thanks for reaching out! Honestly, this is sloppy package development from me. I have made no effort to check whether it's compatible with previous versions. I don't know off the top of my head whether I have used any features that specifically require v0.25 vs v0.22.

I have let a bunch of packaging issues accumulate so hopefully will fix them all in one fell swoop some time this week.

Best,
Robert
